clients/src/main/resources/common/message/ShareFetchRequest.json

```diff
@@ -32,6 +33,8 @@
       "about": "The current share session epoch: 0 to open a share session; -1 to close it; otherwise increments for consecutive requests." },
     { "name": "MaxWaitMs", "type": "int32", "versions": "0+",
       "about": "The maximum time in milliseconds to wait for the response." },
+    { "name": "AcquireMode", "type": "string", "versions": "2+",
+      "about": "The acquire mode to control max fetch records behavior" },
```

Comment:
Shall we please remove client side changes from the PR and let's have only server side changes as that not requires a KIP.